src/LegendDSP.jl

```diff
@@ -40,12 +40,15 @@ include("saturation.jl")
 include("interpolation.jl")
 include("intersect_maximum.jl")
 include("dsp_sipm.jl")
-include("dsp_sipm_optimization.jl")
```

Comment:
Why this?

Reply:
could be an artifact from main?

Reply:
This was introduced on `dev` 3 weeks ago in commit https://github.com/legend-exp/LegendDSP.jl/commit/4d2c0dc26f04c4aaf09cbb8c2b712971ef05c85b and should be kept